pkg/resources/usercluster/deployment.go

```diff
@@ -68,6 +68,7 @@ type userclusterControllerData interface {
 	KubermaticDockerTag() string
 	GetCloudProviderName() (string, error)
 	UserClusterMLAEnabled() bool
+	IsKonnectivityEnabled() bool
```

Comment:
Implementation of this method is missing in this pull request. Was it added before?

Reply:
It was already there just wasn't included in the interface.